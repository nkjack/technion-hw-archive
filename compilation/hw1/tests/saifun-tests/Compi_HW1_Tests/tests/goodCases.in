/*

sads
*/
/**/
helloWorld
-dash_dsds
-dsa------sdfsdf11r3r-
#sdBdfdsf
#1231234
#-Tfsdf
#23423423_sdfdsfs
@import
!important
! imPOrTaNt
!
	iMpoRTaNT
asdsa : asdsa
>
x + 3 = 5;
+
~
:
/*

		s
	*/
;
{
}
[
]
=
*
.
/*


sfsd
