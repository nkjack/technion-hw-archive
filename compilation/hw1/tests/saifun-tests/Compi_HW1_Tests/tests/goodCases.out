4 COMMENT 4
5 COMMENT 1
6 NAME helloWorld
7 NAME -dash_dsds
8 NAME -dsa------sdfsdf11r3r-
9 HASHID #sdBdfdsf
10 HASHID #1231234
11 HASHID #-Tfsdf
12 HASHID #23423423_sdfdsfs
13 IMPORT @import
14 IMPORTANT !important
15 IMPORTANT ! imPOrTaNt
17 IMPORTANT !
	iMpoRTaNT
18 NAME asdsa
18 COLON :
18 NAME asdsa
19 COMB >
20 NAME x
20 COMB +
20 NUMBER 3
20 EQUAL =
20 NUMBER 5
20 SEMICOLON ;
21 COMB +
22 COMB ~
23 COLON :
27 COMMENT 4
28 SEMICOLON ;
29 LBRACE {
30 RBRACE }
31 LBRACKET [
32 RBRACKET ]
33 EQUAL =
34 ASTERISK *
35 DOT .
Error unclosed comment
